lib/private/Preview/Movie.php

```diff
@@ -92,10 +97,109 @@ public function getThumbnail(File $file, int $maxX, int $maxY): ?IImage {
 				break;
 			}
 		}
-
 		return $result;
 	}
 
+	private function getSparseFile(File $file, int $size): string|false {
+		if ($size >= $file->getSize()) {
+			return false;
+		} else {
```

Comment:
```suggestion
		}
```
To save some indentation, since the previous if-clause always returns.